gradle/buildReceipt.gradle

```diff
@@ -14,8 +14,6 @@
  * limitations under the License.
  */
 
-import org.gradle.internal.os.OperatingSystem
-
 ext {
```

Comment:
Move readBuildReceipt to buildSrc (e.g., `BuildReceiptUtils.readBuildReceipt(File)`) and get rid of the rest of `gradle/buildReceipt.gradle`?